crs_motion_planning/src/path_planning_utils.cpp

```diff
@@ -532,23 +644,23 @@ bool crsMotionPlanner::generateFreespacePlans(pathPlanningResults::Ptr& results)
         results->msg_out = "Failed to perform trajopt on first freespace motion";
         return false;
       }
-
-      curr_joint_traj.header.frame_id = config_->world_frame;
-      // Modify time
-      if (config_->use_gazebo_sim_timing)
-      {
-        std::cout << "MODIFYING TRAJECTORY TIME OUTPUT" << std::endl;
-        for (int i = 0; i < curr_joint_traj.points.size(); ++i)
-        {
-          curr_joint_traj.points[static_cast<size_t>(i)].time_from_start.sec = 0;
-          curr_joint_traj.points[static_cast<size_t>(i)].time_from_start.nanosec = 1e8;
-        }
-      }
     }
     else
     {
       curr_joint_traj = ompl_joint_traj;
     }
+
+    curr_joint_traj.header.frame_id = config_->world_frame;
+    // Modify time
+    if (config_->use_gazebo_sim_timing)
```

Comment:
Can this be configured from a ros parameter ?

Reply:
Not sure what you are directly referring to, but both use_gazebo_sim_timing and world_frame are configurable parameters in the motion_planning_server node.